package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "babel-features",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
You shouldn't bump the version in a PR. That is up to the maintainer to do. You also bumped the wrong version. This is a new feature and should be a minor release.